docs/api.asciidoc

```diff
@@ -185,8 +185,49 @@ import elasticapm
 elasticapm.set_transaction_data({'billing_amount': product.price * item_count})
```

Comment:
Is it really still `set_transaction_data` and not `set_context`?

Reply:
great catch!